src/interface/others/AverageTargetsHit.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class AverageTargetsHit extends React.PureComponent {
+  static propTypes = {
+    casts: PropTypes.string.isRequired,
+    hits: PropTypes.string.isRequired,
+    approximate: PropTypes.bool,
+  };
+
+  render() {
+    const { casts, hits, approximate } = this.props;
+    const averageHits = (hits / casts).toFixed(1);
```

Comment:
One last thing I forgot - please check for 0 casts here
```suggestion
    const averageHits = ((hits / casts) || 0).toFixed(1);
```